src/components/AddBook.js

```diff
@@ -30,26 +37,35 @@ const AddBook = () => {
   };
 
   return (
-    <form onSubmit={handleSubmit}>
-      <input
-        type="text"
-        placeholder="Book title"
-        ref={(node) => { title = node; }}
-      />
-      <input
-        type="text"
-        placeholder="Book author"
-        ref={(node) => { author = node; }}
-      />
-      <select ref={(node) => { category = node; }}>
-        <option value="action">Action</option>
-        <option value="scifi">Science Fiction</option>
-        <option value="fantasy">Fantasy</option>
-      </select>
-      <button type="submit">
-        ADD BOOK
-      </button>
-    </form>
+    <div className={styles.container}>
+      <h2>ADD BOOK</h2>
+      <form onSubmit={handleSubmit}>
+        <input
+          type="text"
+          placeholder="Book title"
+          ref={(node) => {
+            title = node;
+          }}
+        />
+        <input
+          type="text"
+          placeholder="Book author"
+          ref={(node) => {
+            author = node;
+          }}
+        />
+        <select
+          ref={(node) => {
+            category = node;
+          }}
+        >
+          <option value="action">Action</option>
+          <option value="scifi">Science Fiction</option>
+          <option value="fantasy">Fantasy</option>
+        </select>
+        <button type="submit">ADD BOOK</button>
+      </form>
```

Comment:
- [ ] Dear @yuvenalmash, For your form I suggest you make it stretch to the whole page as shown in the Zeplin design, also the border should be a bit lighter.

### Zeplin Design
![Screenshot 2023-01-28 173610](https://user-images.githubusercontent.com/69077061/215272862-80ddefd6-f6a6-4f26-944a-b23a473b8d9c.png)
### Your design
![Screenshot 2023-01-28 173627](https://user-images.githubusercontent.com/69077061/215272864-3b5b82f2-144b-407c-a113-70072edf238d.png)